docs/quick-start.md

```diff
@@ -0,0 +1,220 @@
+# gRPC Swift Quick Start
```

Comment:
Should we mention this example in the README?

Reply:
We should, but I'm going to leave that until I've done the route guide tutorial (in progress) as well, I think it'll make the README a little clear (i.e. here's a quick start guide, here's a more in depth guide). WDYT?

Reply:
SGTM